src/util/psbt/mod.rs

```diff
@@ -119,8 +115,11 @@ impl PartiallySignedTransaction {
         Ok(())
     }
 
-    /// Create a PartiallySignedTransaction from an unsigned transaction, error
-    /// if not unsigned
+    /// Creates a PSBT from an unsigned transaction.
+    ///
+    /// # Errors
+    ///
+    /// If transactions is not unsigned.
     pub fn from_unsigned_tx(tx: Transaction) -> Result<Self, Error> {
```

Comment:
Aside. As noted elsewhere, If we are consuming a value, we should return it in the `Error`. Perhaps a good first issue